src/HttpExtractor.php

```diff
@@ -33,10 +33,22 @@ public function __construct(
 
     public function extract(UriInterface $httpSource, string $filesystemDestination): void
     {
+        $options = $this->getRequestOptions();
+        $options['sink'] = $filesystemDestination;
         try {
-            $requestOptions = $this->getRequestOptions();
-            $requestOptions['sink'] = $filesystemDestination;
-            $this->client->get($httpSource, $requestOptions);
+            $this->sendRequest($httpSource, $options);
+        } catch (EncodingException $e) {
+            // Try to download without content encoding.
+            // Server send invalid Content-Encoding, eg. UTF-8, valid are: gzip, deflate, ...
+            $options['decode_content'] = false;
```

Comment:
V starsich verziach Guzzle sa tato chyba ignorovala, ... preto musime chybu odchytit a spravit novy request, ktory nebude dekodovat obsah response. Nemozeme to uplne vyplnut, lebo vy to zase neslo inym klientom.